src/neural/network_random.cc

```diff
@@ -31,6 +31,12 @@ class RandomNetworkComputation : public NetworkComputation {
     std::uint64_t hash = 0;
     for (const auto& plane : input) {
       hash = HashCat({hash, plane.mask});
+      // This assumes value is not large and not very granular, which should
+      // be a reasonable assumption for a NN input.
+      std::uint64_t value_hash = 
```

Comment:
What about `reinterpret_cast<std::uint32_t>(plane.value)`? :)

Reply:
Sounds better - I'll do that.

Reply:
Done.